controllers/comments.controller.js

```diff
@@ -10,4 +10,27 @@ exports.fetchCommentsByArticleId = (req, res, next) => {
     .catch(err => {
         next(err);
     })
+}
+
+exports.postCommentByArticleId = (req, res, next) => {
+    const comment = req.body
+    
+    const {article_id} = req.params
+    if(!comment.hasOwnProperty('username') && !comment.hasOwnProperty('body')) {
+        return res.status(400).send({msg: 'Body is Malformed'})
+        
+
+    } 
+    if (typeof comment.username !== "string" || typeof comment.body !== "string") {
+        return res.status(400).send({msg: 'Schema Validation Failed'})
+    }
```

Comment:
This logic is fine.
But really we should have this logic in your models, and not directly invoking res from the controller, using promise.reject instead.